### PR TITLE
Fix/utility namespace

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

File: dist/js/common-header.js [hunk 1]
```diff
@@ -11886,5 +11886,5 @@ angular.module("risevision.common.i18n", ["pascalprecht.translate", "risevision.
   $translateProvider
     .determinePreferredLanguage()
     .fallbackLanguage("en")
-    .useSanitizeValueStrategy("sceParameters");
+    .useSanitizeValueStrategy(null);
```
Comment (RV-Adrian) [inline dist/js/common-header.js]: @alexdeaconu @ezequielconte @franciscovallarino
One more PR to reinstate Alex's sanitize fix in CH. Thanks!

Comment (RV-Adrian) [inline dist/js/common-header.js]: @alexdeaconu @ezequielconte @franciscovallarino
I've updated the release number and refreshed stage-2. Thanks!

**Comment by ezequielc** [comment]
@RV-Adrian Is this staged somewhere? Thanks

**Comment by RV-Adrian** [comment]
@ezequielc https://apps-stage-2.risevision.com/

File: dist/js/common-header.js (hunk 1 above)
Comment (alex-deaconu) [inline dist/js/common-header.js]: @RV-Adrian you're missing the dash `-` in your references to me here. My username is @alex-deaconu . Thanks.

**Comment by alex-deaconu** [comment]
@RV-Adrian code looks good 👍 . Waiting for feedback from @ezequielc re stage. Thanks.

**Comment by ezequielc** [comment]
@RV-Adrian I still have the issue: https://apps-stage-2.risevision.com/storage?cid=3f119c3c-afbe-447f-9c9a-330443e55064 Did the stage build worked ok? Thanks

**Comment by RV-Adrian** [comment]
@ezequielc Strange. Yeah, got the green light. I did 'rebuild without cache'. Trying again now!

**Comment by alex-deaconu** [comment]
@RV-Adrian your PR still references the current CH tag, not this branch. Thanks.

**Comment by RV-Adrian** [comment]
@alex-deaconu ah snap! Thanks, updating now
@ezequielc FYI

**Comment by RV-Adrian** [comment]
@alex-deaconu @ezequielc 
Re-staged, please see  https://apps-stage-2.risevision.com/storage?cid=3f119c3c-afbe-447f-9c9a-330443e55064 to confirm

**Comment by ezequielc** [comment]
@RV-Adrian Looks good. Thanks for fixing it. 👍 Cheers!

**Pull request merged**